auth-server/src/main/java/com/fiftyfive/authserver/commons/Constants.java

```diff
@@ -0,0 +1,23 @@
+package com.fiftyfive.authserver.commons;
+
+public class Constants {
+    public static final String CLIENT_ID = "client_id";
+    public static final String CLIENT_SECRET = "client_secret";
+    public static final String GRANT_TYPE = "grant_type";
+    public static final String SCOPE = "scope";
+    public static final String OPENID = "openid";
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
+    public static final String REFRESH_TOKEN = "refresh_token";
+    public static final String TOKEN = "refresh_token";
+    public static final String INVALID_LOGIN = "Login failure";
+    public static final String INVALID_LOGOUT = "Logout failure";
+    public static final String LOGIN_URI = "http://localhost:8180/realms/userdev/protocol/openid-connect/token";
```

Comment:
we can keep these properties in the application.properties file

Reply:
okay ,sure will change.